.github/workflows/ci.yml

```
@@ -92,3 +86,119 @@ jobs:
+        if: steps.cache-mediawiki.outputs.cache-hit != 'true'
+        working-directory: ~
+        run: curl EarlyCopy/.github/workflows/installMediaWiki.sh | bash -s ${{ matrix.mw }} WikibaseLocalMedia
+
+      - uses: actions/checkout@v4
+        with:
+          path: mediawiki/extensions/WikibaseLocalMedia
+
+      - name: Composer install
+        run: composer install --no-progress --no-interaction --prefer-dist --optimize-autoloader
+
+      - run: vendor/bin/phpcs -p -s
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)